doc/sphinx/source/input.rst

```diff
@@ -182,6 +182,8 @@ A list of the datasets for which a cmorizers is available is provided in the fol
 | CERES-SYN1deg                | rlds, rldscs, rlus, rluscs, rlut, rlutcs, rsds, rsdscs, rsus, rsuscs, rsut, rsutcs (3hr)             |   3  | NCL             |
 |                              | rlds, rldscs, rlus, rlut, rlutcs, rsds, rsdt, rsus, rsut, rsutcs (Amon)                              |      |                 |
 +------------------------------+------------------------------------------------------------------------------------------------------+------+-----------------+
+| CLARA-AVHRR                  | clt, clivi, clwvi (Amon)                                                                             |   3  | NCL             |
```

Comment:
I think it should be lwp instead of clwvi, right?

Reply:
Yes, you are right. Forgot to change this after applying the modifications suggested earlier. Thanks for spotting!